.ci/Jenkinsfile

```diff
@@ -12,8 +12,8 @@ pipeline {
     PIPELINE_LOG_LEVEL = "INFO"
     DISPLAY= ":99"
     RELEASE_URL_MESSAGE = "(<https://github.com/elastic/${env.REPO}/releases/tag/${env.TAG_NAME}|${env.TAG_NAME}>)"
-    SLACK_CHANNEL = '#TBD'
-    NOTIFY_TO = 'TBD+release@elastic.co'
+    SLACK_CHANNEL = '#observablt-bots'
+    NOTIFY_TO = 'victor.martinez+release@elastic.co'
```

Comment:
TBD

Reply:
@justinkambic and @kyungeunni, I need your input here, what slack channel and mailing list should we use when the release has done or failed?

Reply:
@v1v I'll message you offline about this.

Reply:
If we update the email group with synthrum@elastic.co we will be good here.